documentation/modules/upgrading/proc-downgrade-cluster-operator.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// assembly-downgrade.adoc
+
+[id='proc-downgrade-cluster-operator-{context}']
+= Downgrading the Cluster Operator to a previous version
+
+If you are encountering issues with Strimzi,
+you can revert your installation.
+
+This procedure describes how to downgrade a Cluster Operator deployment to a previous version.
+
+.Prerequisites
+
+* An existing Cluster Operator deployment is available.
+* You have xref:downloads-{context}[downloaded the installation files for the previous version].
+
+.Procedure
+
+. Take note of any configuration changes made to the existing Cluster Operator resources (in the `/install/cluster-operator` directory).
+Any changes will be *overwritten* by the previous version of the Cluster Operator.
++
+. Revert your custom resources to reflect the supported configuration options available for the version of Strimzi you are downgrading to.
+
+. Update the Cluster Operator.
+
+.. Modify the installation files for the previous version according to the namespace the Cluster Operator is running in.
++
+include::snip-cluster-operator-namespace-sed.adoc[]
++
+.. If you modified one or more environment variables in your existing Cluster Operator `Deployment`, edit the
+`install/cluster-operator/060-Deployment-cluster-operator.yaml` file to use those environment variables.
```

Comment:
In Strimzi 0.19.0, this file is named `050-Deployment-strimzi-cluster-operator.yaml`.

Reply:
Still 060, but new name.